README.md

```diff
@@ -955,6 +955,15 @@ All statements above will return false if used with `===`
 
 ## Our contributors are working on this section. [Would you like to join?](https://github.com/i0natan/nodebestpractices/issues/256)
 
+## ![✔] 7.1. Prefer native JS methods over user-land utils like Lodash
+
+ **TL;DR:** It's often more penalising to use utility libraries like `lodash` and `underscore` over native methods as it leads to unneeded dependencies and slower performance.
+ Bear in mind that with the introduction of the new V8 engine alongside the new ES standards, native methods were improved in such a way that it's now about 50% more performant than utility libraries.
```

Comment:
Can we link to this evidence?

Reply:
Sure, I'll add a link to the spreadsheet containing the data and will try to make the result easier to spot.

Reply:
@Berkmann18 Is this some online blog post or some private Excel file? we have the graph which you show below, maybe link to it?